### PR TITLE
🎨Maintenance: use the same swarm address pools as in osparc-infra

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- This PR mirrors the changes in osparc-infra by @YuryHrytsuk 
- the dns field in the docker-compose files used for manual testing were removed
- NOTE: developers will ensure they delete the docker swarm

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Comment by jsaq007** [comment]
👍🏼

**Review by YuryHrytsuk — APPROVED** [review]
@sanderegg refers to these 2 PRs:
* https://git.speag.com/oSparc/osparc-infra/-/merge_requests/210
* https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/477

**Review by bisgaard-itis — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -396,8 +396,8 @@ leave: ## Forces to stop all services, networks, etc by the node leaving the swa
 
 .PHONY: .init-swarm
 .init-swarm:
-	# Ensures swarm is initialized (careful we use a default pool of 10.20.0.0/16. Ensure you do not use private IPs in that range!)
-	$(if $(SWARM_HOSTS),,docker swarm init --advertise-addr=$(get_my_ip) --default-addr-pool 10.20.0.0/16)
+	# Ensures swarm is initialized (careful we use a default pool of 172.20.0.0/14. Ensure you do not use private IPs in that range!)
+	$(if $(SWARM_HOSTS),,docker swarm init --advertise-addr=$(get_my_ip) --default-addr-pool 172.20.0.0/14)
```
Comment (pcrespov) [inline Makefile:400]: I wonder whether 172.20.0.0/14 should be added as some env var?

**Review by pcrespov — APPROVED** [review]
thx!

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**